src/tests/unit/tests/DetailsView/details-view-content.test.tsx

```diff
@@ -0,0 +1,316 @@
+                isFirstTime: false,
+                enableTelemetry: true,
+                enableHighContrast: false,
+                lastSelectedHighContrast: false,
+                bugService: 'gitHub',
+                bugServicePropertiesMap: { gitHub: { repository: 'gitHub-repository' } },
+            };
+
+            const unifiedScanResultStoreData: UnifiedScanResultStoreData = {
+                results: [],
+                rules: [],
+                targetAppInfo: targetAppInfo,
+                timestamp: timestamp,
+                toolInfo: toolData,
+            };
+
+            const storeMocks = new StoreMocks()
+                .setVisualizationStoreData(visualizationStoreData)
+                .setDetailsViewStoreData(detailsViewState)
+                .setUserConfigurationStoreData(userConfigurationStoreData)
+                .setUnifiedScanResultStoreData(unifiedScanResultStoreData);
+
+            const storesHubMock = createStoresHubMock(storeMocks);
+
+            deps.dropdownClickHandler = dropdownClickHandler.object;
+
+            const props = new DetailsViewContainerPropsBuilder(deps)
+                .setStoreMocks(storeMocks)
+                .setStoresHubMock(storesHubMock.object)
+                .build();
+
+            const state = getState(storeMocks, viewType, rightContentPanelConfig);
+            // testObject.state = state;
+
+            getSelectedDetailsViewMock
+                .setup(gsdvm =>
+                    gsdvm(
+                        It.isObjectWith({
+                            assessmentStoreData: state.assessmentStoreData,
+                            visualizationStoreData: state.visualizationStoreData,
+                            pathSnippetStoreData: state.pathSnippetStoreData,
+                        }),
+                    ),
+                )
+                .returns(() => viewType);
+
+            const cardSelectionViewData: CardSelectionViewData = {} as CardSelectionViewData;
+            getCardSelectionViewDataMock
+                .setup(g =>
+                    g(
+                        state.cardSelectionStoreData,
+                        state.unifiedScanResultStoreData,
+                        isResultHighlightUnavailableStub,
+                    ),
+                )
+                .returns(() => cardSelectionViewData)
+                .verifiable(Times.once());
+
+            const cardsViewData: CardsViewModel = {} as any;
+            getCardViewDataMock
+                .setup(m =>
+                    m(
+                        state.unifiedScanResultStoreData.rules,
+                        state.unifiedScanResultStoreData.results,
+                        cardSelectionViewData,
+                    ),
+                )
+                .returns(() => cardsViewData);
+
+            const rendered = shallow(<DetailsViewContent {...props} />);
+            expect(rendered.debug()).toMatchSnapshot();
```

Comment:
We should be ideally using `getElement` over `debug`. If you're getting errors due to type proxy mocks, use actual type mocks.